TrafficCapture/trafficCaptureProxyServer/src/main/java/org/opensearch/migrations/trafficcapture/proxyserver/Main.java

```diff
@@ -40,13 +38,14 @@ private static Settings getSettings(@NonNull String configFile) {
         return builder.build();
     }
 
-    private static IConnectionCaptureFactory getConnectionCaptureFactory(String traceLogsDirectory) {
-        return new FileConnectionCaptureFactory(traceLogsDirectory);
+    private static IConnectionCaptureFactory getConnectionCaptureFactory(String kafkaPropsPath) throws IOException {
+        //return new FileConnectionCaptureFactory("./traceLogs");
+        return new KafkaCaptureFactory(kafkaPropsPath);
     }
 
     public static void main(String[] args) throws InterruptedException, IOException {
 
-        String traceLogsDirectory = args[0];
+        String kafkaPropsPath = args[0];
```

Comment:
We really need to invest in a command line parser for the short term so that we can easily keep the file logging in place (though it may not stay long term).  Longer term, all of the offloader libraries should be implemented as ServiceProviders.

Reply:
Agreed